itests/standalone/basic/src/test/java/org/wildfly/camel/test/cxf/rs/CXFRSConsumerIntegrationTest.java

```diff
@@ -41,6 +41,7 @@
 import org.wildfly.extension.camel.CamelAware;
 import org.wildfly.extension.camel.CamelContextRegistry;
 
+@org.junit.Ignore("https://github.com/wildfly-extras/wildfly-camel/issues/2601")
```

Comment:
Just curious, what's wrong with good old non-fully-qualified `@Ignore`? (likewise for the other ignored test).

Reply:
FQ annotation was easier to paste and will be easier to remove once 2601 is fixed.